skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -103,10 +106,28 @@ private void setupAdapter() {
         recyclerView.setAdapter(formAdapter);
     }
 
-    @OnClick(R.id.bViewWifi)
-    public void viewWifiNetworks() {
-        Intent intent = new Intent(this, WifiActivity.class);
-        startActivity(intent);
+    @OnClick(R.id.bReceiveForms)
+    public void chooseReceivingMethods() {
+        Intent hotspotIntent = new Intent(this, HpReceiverActivity.class);
```

Comment:
Same as mentioned in FilledFormsFragment.

Reply:
Fixed.